app/config.py

```diff
@@ -1 +1,12 @@
+import os
+
 DEBUG = True
+
+DEFAULT_RENDERERS = ['app.helper.custom_api_render.JSONRenderer',
```

Comment:
GREAT CATCH, completely forgot about that, had to use a custom renderer when I started using the the Flask API package as it automatically jsonify's objects and I was returning a dict with a datetime which we know the jsonifier cannot handle, so I wrote the custom handler and some code to handle it, did the fix and forgot why I did the fix in the first place by not removing that. Good job!